blogposts/2024-05-15-improved-gallery-fabric-0.74.md

```diff
@@ -0,0 +1,103 @@
+---
+post_title: "RNW 0.74 Launches: From Gallery Enhancements to Fabric Exploration!"
```

Comment:
```suggestion
post_title: "RNW 0.74 Launches: A Gallery Glow-up and Fabric Foundations!"
```

Maybe a little corny, but I'm reacting to "Fabric Exploration" since that makes it sound like it's just a research project.

Reply:
I love the change to Glow-up!